modules/python/src2/cv2.cpp

```diff
@@ -933,27 +954,6 @@ PyObject* pyopencv_from(const Point3f& p)
 }
 
 template<>
-bool pyopencv_to(PyObject* obj, Vec3d& v, const char* name)
-{
-    (void)name;
-    if(!obj)
-        return true;
-    return PyArg_ParseTuple(obj, "ddd", &v[0], &v[1], &v[2]) > 0;
-}
-
-template<>
-PyObject* pyopencv_from(const Vec3d& v)
-{
-    return Py_BuildValue("(ddd)", v[0], v[1], v[2]);
```

Comment:
@alalek First of all, let me explain what I have done. The removed ones have the format (ddd) or (dd) which is converted into Python [tuples](https://www.tutorialspoint.com/python/python_tuples.htm). However, the binders I have provided converts the data into Numpy arrays which is more flexible and powerful.

Second, although I would like to write some tests to prove that existing Python scripts would not complain from the change, I do not know a good way to show such compatibility. Could you point me out to something related? Or should we keep the old ones and have different behaviors for different vectors?